packages/data/src/redux-store/index.js

```diff
@@ -233,11 +233,24 @@ export default function createReduxStore( key, options ) {
 					selector.registry = registry;
 				}
 				const boundSelector = ( ...args ) => {
+					if (
+						selector.normalizeArgs &&
+						typeof selector.normalizeArgs === 'function' &&
+						args?.length
+					) {
+						args = selector.normalizeArgs( args );
+					}
 					const state = store.__unstableOriginalGetState();
 					return selector( state.root, ...args );
 				};
 
+				// Expose normalization method on the bound selector
+				// in order that it can be called when fullfilling
+				// the resolver.
+				boundSelector.normalizeArgs = selector.normalizeArgs;
```

Comment:
I don't love this but I couldn't see another way to expose the normalization to the resolver.

Reply:
Thanks for the information 🙇